android/templates/module/default/hooks/android-module.js

```diff
@@ -32,7 +32,7 @@ exports.init = function (logger, config, cli, appc) {
 						return file;
 					}
 					dir = path.dirname(dir);
-					return dir != '/' && scan(dir);
+					return dir !== '/' && scan(dir);
 				}(__dirname)),
 				packageJson = require(packageJsonFile),
```

Comment:
<!--
  0 failure: 
  1 warning:  android/templates...
  
  
  DangerID: danger-id-Danger;
  File: android/templates/module/default/hooks/android-module.js;
  Line: 37;
-->

- :warning: android/templates/module/default/hooks/android-module.js line 37 – Found non-literal argument in require (security/detect-non-literal-require)